docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md

```diff
@@ -0,0 +1,131 @@
+# Existing EKS Cluster Open Source Observability Accelerator
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Existing EKS Cluster Open Source Observability pattern, using AWS native tools such as CloudWatch, AWS Managed Prometheus(AMP), AWS Managed Grafana(AMG), and X-Ray and Open Source tools such as Amazon Distro for OpenTelmetry (ADOT) and Prometheus Node Exporter.
+
+![Architecture](../images/CDK_Architecture_diagram.png)
+
+This example makes use of CloudWatch, as a metric and log aggregation layer, while X-Ray is used as a trace-aggregation layer. In order to collect the metrics and traces, we use the Open Source ADOT collector. Fluent Bit is used to export the logs to CloudWatch Logs.
+
+In this architecture, AWS X-Ray provides a complete view of requests as they travel through your application and filters visual data across payloads, functions, traces, services, and APIs. X-Ray also allows you to perform analytics, to gain powerful insights about your distributed trace data.
+
+Utilizing CloudWatch and X-Ray as an aggregation layer allows for a fully-managed scalable telemetry backend. In this example we get those benefits while still having the flexibility and rapid development of the Open Source collection tools.
```

Comment:
Same we are using AMP and AMG for Monitoring. I think copy paste the doc from OSS pattern.

Reply:
Fixed the issues. Please review